app/controllers/untappdAuth.js

```diff
@@ -26,7 +26,7 @@ function untappdAuth(req, res) {
         .then(response => response.json())
         .then(function(data) {
 
-        User.find({
+          User.find({
```

Comment:
Shouldn't you want to throw this in an async function and then wrap this in a try/catch block?